hack/util.sh

```diff
@@ -237,6 +237,26 @@ function tryuntil {
 	done
 }
 
+# tryuntil_failure loops, retrying an action until it fails or times out after 90 seconds.
+function tryuntil_failure {
```

Comment:
Save me the bash read.  Why is this different than `wait_for_command 'os:cmd::expect_failure "command"'` ?